### PR TITLE
DO NOT MERGE: API custom domain

### PR DESCRIPTION
**Pull request by eeshansim**
### PR submission checklist
- [x] Checked all content for grammatical errors (using Grammarly, for example)
- [x] Added new files to `docs/vuepress/sidebar.js`
- [x] Check for broken links
```bash
npm run check
```

### CODE REVIEW

File: docs/api-mgmt/api-endpoints.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+## Creating an Endpoint
+
+Clicking on **Create new endpoint** within the API Collection tab brings up a dialog that allows you to add a new Endpoint to the API Collection. The recipe for this endpoint can be chosen from any callable recipe: it does not have to be in the same folder as other existing collection endpoints.
+
+![Create endpoint dialog](~@img/api-mgmt/add-new-endpoint.png)
+*Create endpoint dialog*
+
+## Editing an Endpoint
+
+The **•••** icon in the upper-right corner of the listed endpoint within the API Collection brings up a menu that allows editing or deleting an individual endpoint within the collection. The editing dialog looks like this:
```
Comment (jandonyada) [inline docs/api-mgmt/api-endpoints.md]: Ellipsis? Or both, i.e. The ellipsis (**•••**) icon

File: docs/api-mgmt/api-collections.md [hunk 2]
```diff
@@ -1,95 +1,80 @@
...
+## Customize URL path for API collection
 
-## Testing an Endpoint
+API collection URL paths are sub-directory paths that you can use to define your API collection. For example, you can use the path prefix to differentiate collections for sales, marketing, or HR.
 
-At the bottom of the endpoint description section, there is a Test button. This can be used to make a REST call to the endpoint for testing purposes.
+Navigate to this page by **selecting your desired API collection** > click on the **Settings** tab, and selecting **URL settings** on the left menu.
 
-The input parameters can be entered in the **Parameters** section of the endpoint description. Frequently the **Example value** shown in the endpoint description will suffice for a test. For example:
+You can edit the URL path for this collection. Workato recommends to a meaningful name and standardizing the name across development teams. Having standardized collection names across your company will simplify the [recipe export process](/recipe-development-lifecycle.md) during versioning changes and updates between development, testing, and production-ready endpoint.
```
Comment (jandonyada) [inline docs/api-mgmt/api-collections.md]: Workato recommends using a meaningful

File: docs/api-mgmt/api-endpoints.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+| Code | Description |
+| ---- | ----------- |
+| 200  | If the call was successful, the response message will be returned with a "success reply". |
+| error codes | If a call to the API fails, then an error status will be returned to the HTTP client. This section documents the possible error codes that might be returned. |
+
+## Testing an Endpoint
+
+To the right of the **Parameters** section, there is a **Try it out** button. This can be used to make a REST call to the endpoint for testing purposes.
+
+The input parameters can be entered in the **Parameters** section of the endpoint description. Frequently the **Example value** shown in the endpoint description will suffice for a test. For example:
```
Comment (jandonyada) [inline docs/api-mgmt/api-endpoints.md]: >Frequently the **Example value**

Often, the **Example value**

File: docs/api-mgmt/api-prefix.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
+---
+title: API Prefix
+date: 2020-01-20 00:00:00 Z
+---
+
+## API Prefix
+
+API prefixes are base paths are unique to each Workato account. You can use it to define your API platform environment. For example, you can use the API prefix to differentiate development, testing, and production callable endpoints. Doing so allows you to reuse your collection paths.
```
Comment (jandonyada) [inline docs/api-mgmt/api-prefix.md]: >API prefixes are base paths are unique

API prefixes are base paths that are unique

File: docs/api-mgmt/api-prefix.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+Having standardized collection names across your company simplifies the [recipe export process](/recipe-development-lifecycle.md). For example, your team can easily export recipes from development account (`/acme-dev/sales-api/`) to the production account `/acme/sales-api/` without worrying about conflicting collection paths. API Prefix lets your work seemlessly between your environments.
```
Comment (jandonyada) [inline docs/api-mgmt/api-prefix.md]: >from development account (`/acme-dev/sales-api/`) to the production account

from the development account (`/acme-dev/sales-api/`) to the production account

File: docs/api-mgmt/api-prefix.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+This API prefix will apply for all API collections and endpoints in this account (individual or team). The default API prefix is set to the initials of your account profile name.
+
+::: warning
+Changes to API prefix will require existing clients of the API to make adjustments on their end.
+:::
+
+### API Prefix for old API collection
+
+All newly created Workato API collections come preconfigured with an API prefix. However, this does not apply to older API collections. These will retain the existing URL until you decide to upgrade.
```
Comment (jandonyada) [inline docs/api-mgmt/api-prefix.md]: >These will retain the existing URL until you decide to upgrade.

Those will retain their existing URLs until you decide to upgrade them.

File: docs/api-mgmt/custom-domain.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+When you use Workato’s API platform, the default domain is workato.com. If you add a custom domain, Workato manages TLS certificates for your APIs to enabled HTTPS for your custom domain. New certificates are created automatically when you add a custom domain. These certificates will automatically renew one month before they expire. Workato uses [Let’s Encrypt](https://letsencrypt.org/) to create and manage TLS certificates for your APIs.
+
+::: tip
+The original API endpoint will continue to work in parallel with your custom domain.
+:::
+
+## How to remove custom domain configuration
+
+You may have to remove the current custom domain configuration if you would like to reconfigure your API platform to a different custom domain. To do this, simply select **Remove** button on the top-right of the screen.
```
Comment (jandonyada) [inline docs/api-mgmt/custom-domain.md]: >To do this, simply select **Remove** button on the top-right of the screen.

To do this, simply click the **Remove** button on the top-right of the screen.

File: docs/api-mgmt/dashboard.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+Monitor the performance of your API platform as a whole, or get granular data about a specific endpoint collection or requesting customer.
+
+**Successful request** and **Errors** provide a big picture view of the health of your API endpoints.
+
+Use the **Policy violations** metric to identify key clients or collections that produce abnormal API calls. See here for more information on [API Access Policies](/api-mgmt/api-access-policies.md).
+
+## Filter parameters
+
+Use filter parameters to modify the dashboard output. By default, the filters are set to `Last 30 days`, `All clients` and `All collections`.
```
Comment (jandonyada) [inline docs/api-mgmt/dashboard.md]: >modify the dashboard output. By default, the filters are set to `Last 30 days`, `All clients` and `All collections`.

Doesn't sound too friendly. How about:

By default, the dashboard is set to `Last 30 days`, `All clients` and `All collections`. You can change the filters to find your preferred dashboard view.

File: docs/api-mgmt/dashboard.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+![API Dashboard Tab](~@img/api-mgmt/api-dashboard.png)
+*API Dashboard Tab*
+
+Monitor the performance of your API platform as a whole, or get granular data about a specific endpoint collection or requesting customer.
+
+**Successful request** and **Errors** provide a big picture view of the health of your API endpoints.
+
+Use the **Policy violations** metric to identify key clients or collections that produce abnormal API calls. See here for more information on [API Access Policies](/api-mgmt/api-access-policies.md).
+
+## Filter parameters
```
Comment (jandonyada) [inline docs/api-mgmt/dashboard.md]: Ultimately this is up to you, but I find that just 'Filters' conveys enough.

File: docs/api-mgmt/dashboard.md (hunk 8 above)
Comment (jandonyada) [reply docs/api-mgmt/dashboard.md]: Might also want to add if the filters persist once users leave the dashboard. That way, people won't be disappointed if they spend a few min fiddling with the filters only to have them reset once they leave the page.

File: docs/api-mgmt/dashboard.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+![API Dashboard filters](~@img/api-mgmt/api-dashboard-filter.png)
+*API Dashboard filters*
+
+| Date filter | Client filter | Collections filter |
+| -- | -- | -- |
+| Select the timeframe that you want to observe.  | Filter by specific clients. | Get more insights by diving into collections and specific endpoints. |
+
+## Top request count
+
+Finally, obtain aggregate data about who is your active API consumers and your most popular endpoint. This display will change according to the filter parameters specified above.
```
Comment (jandonyada) [inline docs/api-mgmt/dashboard.md]: >Finally, obtain aggregate data about who is your active API consumers and your most popular endpoint

Finally, obtain aggregated data about who your most active API consumers are, as well as your most popular endpoints. This display will change according to the filters you specified above.

File: docs/api-mgmt/api-endpoints.md (hunk 1 above)
Comment (eeshansim) [reply docs/api-mgmt/api-endpoints.md]: the icon

Comment (jandonyada) [reply docs/api-mgmt/api-endpoints.md]: I meant, would it be better if we call it the ellipsis icon, i.e.

The ellipsis (**•••**) icon in the upper-right corner....

**Comment by dkogias** [comment]
The new set of (static) IP addresses that will be used for custom domains aren't mentioned, based on a quick scan of this PR.  They are:

```
$ host myendpoint.kogias.org
myendpoint.kogias.org is an alias for apim-custom.workato.com.
apim-custom.workato.com has address 52.203.235.136
apim-custom.workato.com has address 18.211.121.35
apim-custom.workato.com has address 34.232.254.255
```

**Pull request merged**